<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_0">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>451</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CRYPTANALYSIS OF SUBSTITUTION CIPHERS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>291</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>USING GENETIC ALGORITHM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_credits">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Credits</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_help">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cs">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>306</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cryptosystem</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ca">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>260</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cryptanalysis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_plots">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>120</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Statistics</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
